### ISSUE TITLE
Is it possible to render chart in a new window?

### ISSUE THREAD

**Issue by alexgonch** [amcharts/amcharts4#3404]
I have modified [Column chart with axis break](https://www.amcharts.com/demos/column-chart-with-axis-break/) example to attempt to create a chart in a new window: [https://jsfiddle.net/o4n71ca0/](https://jsfiddle.net/o4n71ca0/) (lines 17 to 22, and 24)

Problem is, the `chartdiv-2` stays empty. No errors are thrown.

**Question:** is it possible to make this work and have the chart rendered in a new window?

**Comment by martynasma** [comment]
I don't think it will work like that.

That being said, opening popups is a nasty thing and a subject to a bunch of security restrictions, quite rightfully so.

My two-cent: you should consider creating a modal within the same document as the main app.

**Comment by alexgonch** [comment]
@martynasma thanks for your response!

We are trying to achieve multi-window functionality in our app that is provided by enhanced React Portal (something like [this](https://medium.com/hackernoon/using-a-react-16-portal-to-do-something-cool-2a2d627b0202)). This has proven to work really well so far however we just can't get the charts to render.

I have inspected the code in [Instance.ts](https://github.com/amcharts/amcharts4/blob/master/src/.internal/core/utils/Instance.ts) but don't see anything that could be responsible for the issue we're having. Could you please point me in the right direction?

**Comment by martynasma** [comment]
I'm sorry, I don't have any pointers for you.

**Comment by alexgonch** [comment]
I have deduced that [this line](https://github.com/amcharts/amcharts4/blob/7297b2acabc15a67c1254b7848c3ded3f6bb4c21/src/.internal/core/utils/Instance.ts#L44) returns null despite receiving an HTML element from the new window. The reason is that `instanceof HTMLElement` returns false because basically `window.HTMLElement !== newWindow.HTMLElement`. Not sure what the right fix for this would be.

**Comment by alexgonch** [comment]
For anyone running into the same problem, a little follow-up: 

Initially, I tried overwriting the `getElement()` in my node_modules using [patch-package](https://www.npmjs.com/package/patch-package). This worked and I was able to render the chart but a bunch of its essential features wouldn't work (such as the cursor, timeline, exports, and more), because clearly they've been implemented to only work in the primary window. 

We ended up implementing an embeddable chart in our app and rendering the charts in the new window using an iframe.

Closing this issue.
